src/test/java/org/jenkinsci/plugins/stashNotifier/DescriptorImplTest.java

```diff
@@ -74,20 +77,30 @@ public void testConfigure() throws Descriptor.FormException {
         //given
         doNothing().when(desc).save();
 
+        ServletContext servletContext = PowerMockito.mock(ServletContext.class);
```

Comment:
If possible try not to use PowerMockito but only Mockito. You can have a look in the internet why to avoid PowerMock.

PowerMockito is mainly required to mock constructors and static methods. That's often the case for legacy code that was written without testability in mind. At first sight that's not the case here.